.evergreen/run-tests.sh

```diff
@@ -1,16 +1,14 @@
 #!/bin/sh
 set -o errexit  # Exit the script with error if any of the commands fail
 
-# Supported/used environment variables:
-#   SSL             Set to "yes" to enable SSL. Defaults to "nossl"
-#   MONGODB_URI     Set the suggested connection MONGODB_URI (including credentials and topology info)
-#   TESTS           Optional TESTS environment variable for run-tests.php
-#   API_VERSION     Optional API_VERSION environment variable for run-tests.php
-
-SSL=${SSL:-nossl}
-MONGODB_URI=${MONGODB_URI:-}
-TESTS=${TESTS:-}
-API_VERSION=${API_VERSION:-}
+# Supported environment variables
+API_VERSION=${API_VERSION:-} # Optional API_VERSION environment variable for run-tests.php
+CRYPT_SHARED_LIB_PATH="${CRYPT_SHARED_LIB_PATH:-}" # Optional path to crypt_shared library
+MONGODB_URI=${MONGODB_URI:-} # Connection string (including credentials and topology info)
+SKIP_CRYPT_SHARED="${SKIP_CRYPT_SHARED:-no}" # Specify "yes" to ignore CRYPT_SHARED_LIB_PATH. Defaults to "no"
+SSL=${SSL:-no} # Specify "yes" to enable SSL. Defaults to "no"
+SSL_DIR=${SSL_DIR-} # Optional SSL_DIR environment variable for run-tests.php
```

Comment:
This is technically redundant since `$SSL_DIR` is exported by `$PREPARE_SHELL` in the Evergreen config, but I  appreciated that this lines up with the documented env vars in `basic.inc` and `CONTRIBUTING.md`.